meson.build

```diff
@@ -128,6 +129,15 @@ kyuafile = configure_file(input : 'Kyuafile.in', output : 'Kyuafile', configurat
 test('kyua', kyua_exe, args : ['--config=none', 'test', '--kyuafile', kyuafile, '--build-root', meson.current_build_dir()])
 subdir('tests')
 
+
+with_python = get_option('python') \
+  .require(libtype == 'static', error_message: 'python bindings require static linking') \
+  .require(get_option('debug') == false, error_message: 'python bindings require non debug build type')
```

Comment:
this doesn't sound correct? why would the python bindings need a static libpkgconf? this sounds worse than just using the normal libpkgconf.so that is emitted (or allowing either), and i don't see an actual technical reason for this to be required (aside from actually writing the code to do it)

(the same goes for debug being forbidden, which really only adds -g and friends, no?)

Reply:
For debug, it is because on Windows it would link on python3d.dll (note the d suffix), which is not installed by default. Maybe this isn't a real breaker...

For static, it is because I wasn't able to copy the dll at the right place to be installed correctly by the wheel. Maybe I did something wrong, or maybe it is specific to WIndows...

Reply:
hmm.. generally the wheels don't necessarily contain the .so. i don't know what one gets from pypi, but on distros cffi would load a regular system library in everything i've seen. it would be 'weird' to require the static linkage. so, that does all sound windows specific, for which to best handle you'd want to write it to work with either, so static can be used on windows, and linux people can pick whatever.

for python3d.dll, that is indeed a windows thing. on linux there is no such thing- the debug only implies that you add some debugging flags and whatnot and not that a different python library is linked (which is wholly separate and may or may not have debug, might have them split symbols elsewhere, ..., and be out of control of the build system here at all). so that is only valid when host=windows (or at least not for host=linux).

Reply:
I reworked the meson files a little bit. However, it would require a feature not yet in meson for it to work with shared library on Windows. Other cases should be ok (but not tested...)